po/sv.po

```diff
@@ -207,12 +207,12 @@ msgstr ""
 #: ../data/data.cpp:46
 #, fuzzy
 msgid "Haumea"
-msgstr "Noumea"
+msgstr "Haumea"
```

Comment:
this one is still marked as fuzzy

Reply:
Don't know what fuzzy means here

Reply:
"Don't use, it's probably wrong"

Reply:
Why is it in the code then?

Reply:
automatically substitute by gettext program

Reply:
should those be reverted then?

Reply:
if you fixed a translation you should remove its "fuzzy" line